palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/config/ApplicationConfiguration.java

```diff
@@ -151,7 +151,7 @@ public PolicyService policyService(final PolicyClient policyClient) {
 
     @Bean
     public ResultAggregationService resultAggregationService(final AuditService auditService, final PersistenceLayer persistenceLayer) {
```

Comment:
Can we remove the `auditService` parameter seeing as it is no longer required?

Reply:
removed